build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
We've been preferring Kotlin as the language for new Gradle scripts.  How do you feel about switching?

Reply:
https://blog.jetbrains.com/kotlin/2023/04/kotlin-dsl-is-the-default-for-new-gradle-builds/ gives a good summary of why to prefer Kotlin.